task-29

marsel-code opened the pull request:
Добавил верстку vacancy_item, убрал лишние скобки в Root Activity.

Commit:
task-29
Добавил верстку vacancy_item, убрал лишние скобки в Root Activity.

Fargo02 commented on app/src/main/res/layout/vacancy_item.xml:
```diff
@@ -1,6 +1,65 @@
+        style="@style/VacancyItemTitleTextStyle"
+        android:layout_width="@dimen/mark_0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/mark_8dp"
```
Судя по макету тут должно быть 12dp

marsel-code replied:
Тут исправлю.

Fargo02 commented on app/src/main/res/layout/vacancy_item.xml:
```diff
@@ -1,6 +1,65 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="wrap_content"
+    android:padding="@dimen/mark_8dp">
```
![Снимок экрана 2024-11-13 115133](https://github.com/user-attachments/assets/d6ef60bb-5e37-426b-b84c-9cf152f6ad7e)

Сверху и снизу по 9dp, по сути не критично, но если есть время можешь заменить

marsel-code replied:
мне ревью замечание сделал что лучше использовать в коде размеры, кратные 4 или 8. Документация - https://material.io/design/layout/spacing-methods.html#baseline-grid

sashkir7 replied:
Ага, правильно написали. Чтобы на всех устройствах картинка выглядела збс

Fargo02 commented on app/src/main/res/layout/vacancy_item.xml:
```diff
@@ -1,6 +1,65 @@
+    android:layout_height="wrap_content"
+    android:padding="@dimen/mark_8dp">
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/left_guideline"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_begin="@dimen/mark_8dp" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/right_guideline"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_end="@dimen/mark_8dp" />
```
Судя по макету тут по 16dp должно быть, если я не ошибаюсь

marsel-code replied:
у ConstraintLayout padding="@dimen/mark_8dp"> + здесь 8dp получается 16

Fargo02 replied:
![хмм47](https://github.com/user-attachments/assets/c3bb7793-a21a-4a09-9f22-caf3c185b94a)

На самом деле студия странное отображает относительно чего Guideline расположение. Пишет что там от края до линии 8, хотя по факту 16

Commit:
task-29
Добавил верстку vacancy_item, убрал лишние скобки в Root Activity. Исправил замечания.

Fargo02 reviewed (approved):
Гуд
